Lib/sqlite3/test/backup.py

```diff
@@ -0,0 +1,127 @@
+import sqlite3 as sqlite
+from tempfile import NamedTemporaryFile
+import unittest
+
+class BackupTests(unittest.TestCase):
```

Comment:
There should be a `@unittest.skipIf(...)` decorator to skip the whole test case if SQLite version is not 3.6.11 or newer.